src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -16,6 +16,9 @@ namespace NuGet.Packaging
 {
     public static class PackageExtractor
     {
+        private static Lazy<RepositorySignatureInfoProvider> _repositorySignatureInfoProvider =
```

Comment:
Why do you need a lazy field?  Just use `RepositorySignatureInfoProvider.Instance` [here](https://github.com/NuGet/NuGet.Client/pull/2139/files#diff-e71565361fa493b38ca90727d1882fceR1020).